packages/client/__tests__/AuthenticationPasswordReset.test.tsx

```diff
@@ -179,6 +182,154 @@ const passwordResetMachine =
                         },
                     },
                 },
+                'Rendering password reset code screen': {
+                    meta: {
+                        test: async ({ screen }: TestingContext) => {
+                            await waitFor(() => {
+                                const passwordResetCodeScreenContainer =
+                                    screen.getByTestId(
+                                        'password-reset-confirmation-token-screen-container',
+                                    );
+                                expect(
+                                    passwordResetCodeScreenContainer,
+                                ).toBeTruthy();
+                            });
+                        },
+                    },
+                    initial:
+                        'Displaying password reset successful request toast',
+                    states: {
+                        'Displaying password reset successful request toast': {
```

Comment:
Question: why are we expecting a toast when entering the password reset workflow ?

Reply:
I added a comment to explain the reason: https://github.com/AdonisEnProvence/MusicRoom/pull/553/commits/21dee2f90a6ebf7c1d3b40e4610ea61ce39d1cc7#diff-4982ec3e27a3d7f1d2d966a855028736a70f7956b076bf673165d46e9ebc89bcR202.

Please note the state is named _Password reset **code** form_. In this step we want to get the confirmation _code_.